redbot/cogs/downloader/downloader.py

```diff
@@ -583,23 +583,30 @@ async def _repo_add(
             if repo.install_msg:
                 await ctx.send(repo.install_msg.replace("[p]", ctx.clean_prefix))
 
-    @repo.command(name="delete", aliases=["remove", "del"])
-    async def _repo_del(self, ctx: commands.Context, repo: Repo) -> None:
+    @repo.command(name="delete", aliases=["remove", "del"], require_var_positional=True)
+    async def _repo_del(self, ctx: commands.Context, *repos: Repo) -> None:
         """
-        Remove a repo and its files.
+        Remove repos and their files.
 
-        Example:
+        Examples:
             - `[p]repo delete 26-Cogs`
+            - `[p]repo delete 26-Cogs Laggrons-Dumb-Cogs`
 
         **Arguments**
 
-        - `<repo>` The name of an already added repo
+        - `<repos...>` The repo or repos to remove.
         """
-        await self._repo_manager.delete_repo(repo.name)
+        for repo in set(repos):
+            await self._repo_manager.delete_repo(repo.name)
 
-        await ctx.send(
-            _("The repo `{repo.name}` has been deleted successfully.").format(repo=repo)
-        )
+        if len(repos) == 1:
+            await ctx.send(
+                _("The repo `{repo.name}` has been deleted successfully.").format(repo=repo)
+            )
+        else:
+            await ctx.send(
+                _("Successfully deleted repos: ") + humanize_list([i.name for i in set(repos)])
+            )
```

Comment:
I'm not sure whether I managed to be 100% consistent but I tried my best to have both the singular and plural forms look similar so that people can parse through it faster knowing the consistent look the message has.
Here's what I would suggest to use instead:
```suggestion
        await ctx.send(
            (
                _("Successfully deleted repos: ")
                if len(repos) > 1
                else _("Successfully deleted the repo: ")
            ) + humanize_list([i.name for i in set(repos)])
        )
```